content/blog/2021-12-15-react-conf-2021-recap.md

```diff
@@ -0,0 +1,57 @@
+
+**Some quick stats:**
+
+* We streamed the event twice to accommodate community members in different time zones. 6,900 viewers for the Live event and 5,800 viewers for the Replay event ten hours after the Live event.
+* 61,000 people watched the talks after the event on [YouTube](https://www.youtube.com/channel/UC1hOCRBN2mnXgN5reSoO3pQ/videos).
+* 5,700 members participated in the live discussion via our [Discord server](https://discord.gg/JVtWJyagJw).
+* 8 languages were officially supported with professional video captioning and multilingual moderators to accommodate the global React community. React Conf was available in Arabic, Spanish, Hindi, Japanese, Portuguese, Russian, Chinese, and English.
+* A team of moderators were brought in to enforce the [Code of Conduct](https://github.com/facebook/react/blob/main/CODE_OF_CONDUCT.md) at the event.
+
+In previous years, we've used the React Conf stage to deliver industry changing announcements such as [React Native](https://engineering.fb.com/2015/03/26/android/react-native-bringing-modern-web-techniques-to-mobile/) and [React Hooks](https://reactjs.org/docs/hooks-intro.html). This year, we shared our multi-platform vision for React, starting with the release of **React 18** and the **concurrent rendering engine**.
+
+After [piloting React 18](https://engineering.fb.com/2020/05/08/web/facebook-redesign/) with select teams at Meta, and are excited to announce that we're ready to share React 18 with a wider audience. We are releasing the React 18 RC to open source so that members of the React community can begin using the new concurrent rendering engine today.
+
+This release is a milestone of years of collaborative research between the React, React Native, and React Data teams. With this release we're rolling out features that power the [facebook.com](https://www.facebook.com/) site and set the foundation for upcoming features such as **Server Components** and **React Native pre-rendering**.
+
+We also shared how we're raising the bar across many platforms with our [many platform vision](https://reactnative.dev/blog/2021/08/26/many-platform-vision). The foundation set by React 18 will power our investment into platforms such as Desktop and VR.
+
+![A diagram showing the proposed platforms React plans to support](../images/blog/react-conf-2021-recap/multi-platform-vision.png)
+
+## Here are some key highlights from the conference {#key-highlights-from-conf}
+
+* **React 18 RC:** We announced React 18, the foundation for the future of React. This release includes the **new concurrent renderer** that powers the [facebook.com](http://facebook.com/) site, **Suspense** , and many upcoming features such as **Server Components** , and **React Native pre-rendering**. Learn more at [Andrew Clark](https://twitter.com/acdlite) and [Juan Tejada's](https://twitter.com/_jstejada) talk in the [React 18 Keynote](https://www.youtube.com/watch?v=FZ0cG47msEk).
```

Comment:
Here too it’s easy to misunderstand it as “this release includes […] server components”